doc/manual/getting_started_core.rst

```diff
@@ -176,6 +176,25 @@ Within a parallel block, some statements can be made sequential again using a ``
 
 .. _rtio-analyzer-example:
 
+A for or while loop inside of a parallel block implicitly shifts to sequential execution of sub-statements, since the for loop is considered the top-level statement.
```

Comment:
"since the <s>for</s> loop..."

Reply:
This holds true for ANY subroutine, not just a for or while loop.  We should say that all top-level statements within a parallel block are executed in parallel, but the parallel execution is NOT inherited by any sub-statements.